tests/installation/livecd_network_settings.pm

```diff
@@ -17,8 +17,8 @@ use testapi;
 # 'network settings' after welcome
 sub run() {
     assert_screen 'inst-network_settings-livecd';
-    # ne'x't
-    send_key 'alt-x';
+    # 'n'ext
+    send_key 'alt-n';
```

Comment:
yes, it looks like there are many more occassions of alt-n which should be $cmd{next}. I will try to change all of these and provide it in an explicit commit.